taxonomies/additives.txt

```diff
@@ -139,7 +139,7 @@ efsa_evaluation_url:en: http://dx.doi.org/10.2903/j.efsa.2013.3357
 efsa_evaluation_date:en: 2013-10-22
 efsa_evaluation:en: Scientific Opinion on the re-evaluation of riboflavin (E 101(i)) and riboflavin-5'-phosphate sodium (E 101(ii)) as food additives.
 additives_classes:en: en:colour
-vegan:en:yes
```

Comment:
What is wrong with E101. The dutch vegetarian society sees no issue.

Reply:
It's listed as may not be vegan/vegetarian by the UK vegetarian society: "Found in green vegetables, milk, eggs and yeast. Can be produced commercially from micro-organisms, such as yeast. Used to create a bright lemon yellow food colour. Found in salad dressings, confectionery, tablet coatings and powdered drinks."

Reply:
Foodinfo.net: E101 can be consumed by all religious groups, vegans and vegetarians. Although it can be produced from milk, this is not done commercially.